en/docs/Learn/APIMonetization/monetizing-an-api.md

```diff
@@ -0,0 +1,703 @@
+
+1.  Enable WSO2 API Manager Analytics.  
+    
+     You need to enable WSO2 API Manager Analytics in order to use Monetization, because when working with Monetization the API usage is derived from WSO2 API Manager Analytics.  
+     
+     <html>
+     <p>
+     Copy the <a href="https://raw.githubusercontent.com/wso2/analytics-solutions/master/features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/monetization-summary-siddhi-file/APIM_MONETIZATION_SUMMARY.siddhi">APIM_MONETIZATION_SUMMARY.siddhi</a>
+    file and add it in to the <code><API-M-ANALYTICS_HOME>/wso2/worker/deployment/siddhi-files</code> directory if it is not already available.
+    </p>
+    </html>
+
+2.  Configure the workflows.  
+
+     You need to do this in order to ensure that the correct workflows are engaged when a subscription is added or removed.
+
+     1.  Sign in to the WSO2 API-M management console.
+     2.  Edit the workflow executors in the `workflow-extensions.xml` (`/_system/governance/apimgt/applicationdata/workflow-extensions.xml`) file.
+        
+    ``` xml tab="Format"
+    <SubscriptionCreation executor="<billing-engine-related-SubscriptionCreationWorkflowExecutor>"/>
+    <SubscriptionDeletion executor="<billing-engine-related-StripeSubscriptionDeletionWorkflowExecutor>"/> 
+    ```
+
+    ``` xml tab="Example"
+    <SubscriptionCreation executor="org.wso2.apim.monetization.impl.workflow.StripeSubscriptionCreationWorkflowExecutor"/>
+    <SubscriptionDeletion executor="org.wso2.apim.monetization.impl.workflow.StripeSubscriptionDeletionWorkflowExecutor"/>
+    ```
+
+#### Step 1 - Create a subscription tier
+
+1.  Navigate to the WSO2 API Manager admin dashboard.  
+     
+     `https://<API-M_host>:<port>/admin`
+
+2.  Create a subscription policy.  
+     
+     Specify the monetization plan and other relevant data based on your preference. For more information, see [Adding a new subscription-level throttling tier](../../Learn/RateLimiting/adding-new-throttling-policies#Adding-a-new-subscription---level-throttling-tier).
+
+     <html>
+      <div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>When using Stripe as a billing engine it only allows you to create monetization plans for commercial tiers. Therefore, as Stripe is the billing engine in this example, make sure to create a subscription policy that has a commercial tier.</p>
+      </div> 
+     </html>
+
+
+     <html>
+         <head>
+         </head>
+         <body>
+             <img src="../../assets/img/Learn/subscription-commercial-tier.png" alt="Subscription to a commercial tier" title="Commercial tier subscription" width="400" />
+         </body>
+     </html>
+
+     After you save the policy, a plan gets created in the Stripe account of the tenant admin.  
+
+     ![Stripe account after creating a commercial tier](../../assets/img/Learn/Stripe-account-after-creating-a-commercial-tier.png)  
```

Comment:
Imge path needs to be corrected

Reply:
Fixed. 
I added the path based on the example on page 6 in the "Guidelines for Github Documentation" document. Looks like I misinterpreted the example.